broadinstitute/single_cell_portal_core
# disabling gene search bar if study has no genes (SCP-4363)

Disables the search bar and removes gene set upload button (removing it was easier and IMHO cleaner-looking than greying it) if the study has no genes

![image](https://user-images.githubusercontent.com/2800795/172447429-9d284c53-3320-49ae-bbfa-dc7d8b44de86.png)


TO TEST:
1. load a study in the explore tab with genes
2. confirm gene search bar appears as normal
3. load a study with clusters but no genes
4. confirm search bar is disabled
